docs/GettingStartedDocs/UseNonDCAPDriverInAcc.md

```diff
@@ -0,0 +1,53 @@
+# Using the Non-DCAP Driver in an ACC VM
+
+ACC VMs come pre equipeed with the azure DCAP stack preinstalled. This can interfere with non-azure dcap clients, 
+so in this case we must disable the dcap driver.
+
+To do so, first lets get the non-dcap driver (if we haven't already).
+```
+curl https://download.01.org/intel-sgx/sgx-linux/2.9/distro/ubuntu16.04-server/sgx_linux_x64_driver_2.6.0_95eaa6f.bin --output sgx_linux_x64_driver_2.6.0_95eaa6f.bin
```

Comment:
The title of the doc says "ACC VM", but this line is Linux specific.  Does that mean there are no Windows ACC VM's?